code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/ServiceDiscoveryServer.java

```diff
@@ -0,0 +1,17 @@
+package io.cattle.platform.servicediscovery.service;
+
+import io.cattle.platform.core.model.Service;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.List;
+import java.util.Map;
+
+public interface ServiceDiscoveryServer {
```

Comment:
Yeah.  I noticed that too, but I think it's still better.  Not a merge blocker to me though. :-)